nepalingo-web/src/components/GreetingCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+interface GreetingCardProps {
+  message: string;
+}
+
+const cardStyle: React.CSSProperties = {
+  border: '2px solid #000',
+  borderRadius: '15px',
+  padding: '20px',
+  margin: '10px',
+  textAlign: 'center',
+  backgroundColor: '#f0f0f0',
+  boxShadow: '0 4px 8px rgba(0, 0, 0, 0.2)',
+  fontFamily: 'Arial, sans-serif',
+};
+
+const headingStyle: React.CSSProperties = {
+  fontSize: '24px',
+  color: '#333',
+};
+
+const GreetingCard: React.FC<GreetingCardProps> = ({ message }) => {
```

Comment:
This looks like a great start! Let's have it say 'Good Morning, Puka' - you can look at how it's implemented in the home page right now